apps/antalmanac/src/components/RightPane/AddedCourses/CustomEventDetailView.tsx

```diff
@@ -55,30 +37,33 @@ const CustomEventDetailView = (props: CustomEventDetailViewProps) => {
         <Card>
             <CardHeader
                 titleTypographyProps={{ variant: 'subtitle1' }}
-                className={classes.root}
                 title={customEvent.title}
                 subheader={readableDateAndTimeFormat(customEvent.start, customEvent.end, customEvent.days)}
+                style={{ padding: 8 }}
             />
-            <CardActions disableSpacing={true}>
-                <div className={classes.colorPicker}>
-                    <ColorPicker
-                        color={customEvent.color as string}
-                        isCustomEvent={true}
-                        customEventID={customEvent.customEventID}
-                        analyticsCategory={analyticsEnum.addedClasses.title}
-                    />
-                </div>
-                <IconButton
-                    onClick={() => {
-                        deleteCustomEvent(customEvent.customEventID);
-                    }}
-                >
-                    <Delete fontSize="small" />
-                </IconButton>
-                <CustomEventDialog customEvent={customEvent} scheduleNames={props.scheduleNames} />
-            </CardActions>
+            {!isSkeletonMode && (
+                <CardActions disableSpacing={true} style={{ padding: 0, marginTop: -8 }}>
+                    <div>
+                        <ColorPicker
+                            color={customEvent.color as string}
+                            isCustomEvent={true}
+                            customEventID={customEvent.customEventID}
+                            analyticsCategory={analyticsEnum.addedClasses.title}
+                        />
+                    </div>
+                    <IconButton
+                        onClick={() => {
+                            deleteCustomEvent(customEvent.customEventID);
+                        }}
+                        size="large"
+                    >
+                        <Delete fontSize="small" />
+                    </IconButton>
+                    <CustomEventDialog customEvent={customEvent} scheduleNames={scheduleNames} />
+                </CardActions>
+            )}
         </Card>
```

Comment:
Honestly, just for simplicity, it might be easier to have two different returns (or two components) depending on whether it's the skeleton schedule or not.